docs/man_pages/project/creation/init.md

```diff
@@ -0,0 +1,20 @@
+init
+==========
+
+Usage | Synopsis
+---|---
+General | `$ tns init [--path <Directory>] [--force]`
+
+Initializes a project for development. The command prompts you to provide your project configuration interactively and uses the information to create a new package.json file or update the existing one.
+
+### Options
+* `--path` - Specifies the directory where you want to initialize the project, if different from the current directory. The directory must be empty.
+* `--force` - If set, applies the default project configuration and does not show the interactive prompt. The default project configuration targets the latest official runtimes and sets org.nativescript.<folder_name> for application identifier.
```

Comment:
code formatting for org.nativescript... otherwise, the stuff in the triangle brackets is considered an HTML tag and breaks the output